### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by javitocor**
Dear TSE,

In this milestone I accomplished the following requirements:
- Add style to display component
- Add style to button component
- Add props to button component
- Add propTypes to button component
- Add style to button panel component

Best regards

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,75 @@
...
+  font-weight: bold;
+  font-size: 2.8rem;
+  padding: 18px;
+}
+
+.button {
+  width: 25%;
+  color: black;
+  border: 1px solid gray;
+  font-size: 2.4rem;
```
Comment (whiz25) [inline src/index.css]: - Text must be centered. Add the styling.

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,75 @@
...
     monospace;
 }
+
+.App {
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+}
+
+.display {
```
Comment (whiz25) [inline src/index.css:40]: - Display, App and ButtonPanel components must have a display mode of flex. Add styling for ButtonPanel.js component.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray: 
You have done an amazing job in this milestone. Most requirements have been met. You need to fix a few things before we can merge.
Check my comments for suggestions.


### Martin Afani

**Comment by javitocor** [comment]
Added changes requested by TSE

**Review by fatymahmed — APPROVED** [review]
## Hi @javitocor  
### Good job making the requested changes 🎉🎉
## Status: Approved ✔️

**Pull request merged**